sections/testing/usability-testing.tex

```diff
@@ -1 +1,14 @@
 \section{Usability testing}
+Opposed to the previously mentioned tests, usability testing is not directly focused on the functionality and structure of the code, but rather how easy the system is to use by real users.
+Even though the system may seem exceedingly simply to use by the developers, this is rarely the case for the actual users.
+To increase the usability of the system during the development, the development team can make use of the personas defined in \autoref{sec:personas} during the formal review.
+The reviewers can subjectively evaluate the design choices made against the personas and try to conclude whether all the personas would be able to understand the system.
+
+However, this does not solve the problem of the reviewers knowing how the system works, and can easily navigate through it due to their deep understanding of the system.
+
+Instead, it will be beneficial to conduct usability testing, where the testers try out the system under observation.
+\\\\
+With usability testing, it is possible to get feedback on more subjective parts of the system, such as the user interface and whether the errors displayed to the user are useful and understandable without having a degree in software engineering \cite{SoftwareTesting}.
+
+When conducting usability tests, a set of pre-defined are given to the testers one at a time.
```

Comment:
Mangler der ikke et ord efter "a set of pre-defined"?

Reply:
i accidentally a word